mne/time_frequency/psd.py

```diff
@@ -112,7 +113,209 @@ def _check_nfft(n, n_fft, n_overlap):
+        sfreq must also be supplied.
+    fmin : float
+        Min frequency of interest
+    fmax : float
+        Max frequency of interest
+    tmin : float | None
+        Min time of interest
+    tmax : float | None
+        Max time of interest
+    bandwidth : float
+        The bandwidth of the multi taper windowing function in Hz.
+    adaptive : bool
+        Use adaptive weights to combine the tapered spectra into PSD
+        (slow, use n_jobs >> 1 to speed up computation).
+    low_bias : bool
+        Only use tapers with more than 90% spectral concentration within
+        bandwidth.
+    normalization : str
+        Either "full" or "length" (default). If "full", the PSD will
+        be normalized by the sampling rate as well as the length of
+        the signal (as in nitime).
+    picks : array-like of int | None
+        The selection of channels to include in the computation.
+        If None, take all channels.
+    proj : bool
+        Apply SSP projection vectors. If inst is ndarray this is not used.
+    n_jobs : int
+        Number of CPUs to use in the computation.
+    verbose : bool, str, int, or None
+        If not None, override default verbose level (see mne.verbose).
+
+    Returns
+    -------
+    psds : ndarray, shape ([n_epochs], n_channels, n_freqs)
+        The power spectral densities. If Raw is provided,
+        then psds will be 2-D.
+    freqs : ndarray (n_freqs)
+        The frequencies.
+    """
+    # Prep data
+    data, sfreq = _check_psd_data(inst, tmin, tmax, picks, proj)
+    psds, freqs = _psd_multitaper(data, sfreq, fmin=fmin, fmax=fmax,
+                                  bandwidth=bandwidth, adaptive=adaptive,
+                                  low_bias=low_bias,
+                                  normalization=normalization,  n_jobs=n_jobs,
+                                  verbose=verbose)
+    return psds, freqs
+
+
+def _psd_multitaper(x, sfreq, fmin=0, fmax=np.inf, bandwidth=None,
+                    adaptive=False, low_bias=True, normalization='length',
+                    n_jobs=1, verbose=None):
+    """Helper function for calculating Welch PSD."""
+    from .multitaper import multitaper_psd
+    dshape = x.shape[:-1]
+    n_times = x.shape[-1]
+
+    # This will return the same object if len(dshape) == 1
+    x = x.reshape(-1, n_times)
+
+    # Stack data so it's treated separately
+    psds, freqs = multitaper_psd(x=x, sfreq=sfreq, fmin=fmin, fmax=fmax,
+                                 bandwidth=bandwidth, adaptive=adaptive,
+                                 low_bias=low_bias,
+                                 normalization=normalization, n_jobs=n_jobs,
+                                 verbose=verbose)
+
+    # Combining/reshaping to original data shape
+    n_freqs = len(freqs)
+    psds = psds.reshape(np.hstack([dshape, n_freqs]))
+    return psds, freqs
+
+
 @verbose
+@deprecated('This will be deprecated in release v0.13, see psd_ functions.')
```

Comment:
I was thinking that since we released 0.11, we'd use 0.12 to tell people it will be deprecated, and then actually do it on 0.13. If that's not the typical deprecation cycle I can change it to 0.12